3rdParty/fuerte/src/AsioSockets.h

```diff
@@ -34,8 +34,16 @@ template <typename SocketT, typename F>
 void resolveConnect(detail::ConnectionConfiguration const& config,
                     asio_ns::ip::tcp::resolver& resolver, SocketT& socket,
                     F&& done) {
-  auto cb = [&socket, done(std::forward<F>(done))](auto ec, auto it) mutable {
+  auto cb = [&socket, done = std::forward<F>(done), fail = config._failConnectAttempts > 0](auto ec, auto it) mutable {
```

Comment:
This won't build if `ARANGODB_USE_GOOGLE_TESTS` is not defined, because then we have no member `_failConnectAttempts`
```suggestion
  auto cb = [&socket, done = std::forward<F>(done), fail = config._failConnectAttempts > 0](auto ec, auto it) mutable {
```